ironic.conf.j2

```diff
@@ -66,7 +66,13 @@ send_sensor_data_interval = 160
 bootloader = http://{{ env.IRONIC_URL_HOST }}:{{ env.HTTP_PORT }}/uefi_esp.img
 
 [database]
-connection = mysql+pymysql://ironic:{{ env.MARIADB_PASSWORD }}@localhost/ironic?charset=utf8
+{% if env.MARIADB_TLS_ENABLED == "true" %}
+connection = mysql+pymysql://ironic:{{ env.MARIADB_PASSWORD }}@127.0.0.1/ironic?charset=utf8&ssl=on&ssl_ca=/certs/ca/mariadb/tls.crt
```

Comment:
nit: we could use the env variable to get the certificate path, to have it configured in one place only

Reply:
Fixed! Please take a look again. Thanks